hera_mc/tests/test_sysutils.py

```diff
@@ -15,14 +15,19 @@
 import numpy as np
 
 from .. import (cm_sysutils, cm_partconnect, cm_hookup, cm_utils, utils,
-                cm_sysdef, cm_dossier, cm_active)
+                cm_sysdef, cm_dossier, cm_active, cm_redis_corr)
 
 
 @pytest.fixture(scope='function')
 def sys_handle(mcsession):
     return cm_sysutils.Handling(mcsession)
 
 
+def test_set_redis_cminfo(mcsession):
+    test_out = cm_redis_corr.set_redis_cminfo(redishost=None, session=mcsession, testing=True)
+    assert '{"host": "SNPA000700", "channel": 0}' in test_out['ant_to_snap']
```

Comment:
there are a number of other keys that it doesn't look like you're testing (e.g. cofa)

Reply:
This checks that the dictionary was written to redis.  It doesn't check the data itself.  For instance, I don't even have a 'cofa' per se...

Reply:
This now checks one from each type in the redis_corr module.